streams/src/main/java/org/apache/kafka/streams/processor/internals/SinkNode.java

```diff
@@ -82,7 +103,11 @@ public void process(final Record<KIn, VIn> record) {
 
         final String topic = topicExtractor.extract(key, value, contextForExtraction);
 
-        collector.send(topic, key, value, record.headers(), timestamp, keySerializer, valSerializer, partitioner);
+        final long bytesProduced =
+            collector.send(topic, key, value, record.headers(), timestamp, keySerializer, valSerializer, partitioner);
+
+        bytesProducedSensor.record(bytesProduced, context.currentSystemTimeMs());
```

Comment:
A minor suggestion: I think we can use one sensor that contains two metrics, one as "sum" and one as "count", and then call `record` here once on that sensor which would update both metrics. Similar for consumed.

Examples can be found in those `addInvocationRateAndCountToSensor` etc functions.